.circleci/config.yml

```diff
@@ -78,12 +93,48 @@ jobs:
                --label "build.number=$CIRCLE_BUILD_NUM" \
                --label "build.url=$CIRCLE_BUILD_URL" \
                --label "build.gitref=$CIRCLE_SHA1"
+      - snyk/scan:
```

Comment:
will this block deployments if there are security vulnerabilities? 
If so wouldn't that prevent deploying changes if there are no resolutions for those security vulnerabilities?

Reply:
hey andrew, yes this intentionally blocks deployments with high severity CVEs.

in case we need to go ahead without resolving a vulnerability we can ignore specific issues using the `.snyk` file in this PR (https://github.com/ministryofjustice/community-api/blob/089131e21922be1578dfe23d5c1b7a56c663fbf5/.snyk)